x-pack/plugins/security_solution/public/timelines/components/row_renderers_browser/examples/system_socket.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { createSocketRowRenderer } from '../../timeline/body/renderers/system/generic_row_renderer';
+import { mockEndgameIpv4ConnectionAcceptEvent } from '../../../../common/mock/mock_endgame_ecs_data';
+import { ROW_RENDERER_BROWSER_EXAMPLE_TIMELINE_ID } from '../constants';
+
+const SystemSocketExampleComponent: React.FC = () => {
+  const systemSocketRowRenderer = createSocketRowRenderer({
+    actionName: 'ipv4_connection_accept_event',
+    text: 'accepted a connection via',
```

Comment:
consider using the `i18n` `ACCEPTED_A_CONNECTION_VIA` from `x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/system/translations.ts`